### PR TITLE
Ignore more items of gphdfs regression test results

### PR DESCRIPTION
**Pull request by 50wu**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@50wu Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=1305)!

[Click here](https://cla.pivotal.io/sync/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=1305) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@50wu Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=1305)!

File: gpAux/extensions/gphdfs/regression/init_file [hunk 1]
```diff
@@ -1,4 +1,4 @@
--- start_matchignore
+-- start_distro_specific_ignore
```
Comment (hlinnaka) [inline gpAux/extensions/gphdfs/regression/init_file]: "start_matchignore" is a special keyword for the atmsort.pl script. What is "start_distro_specific_ignore"? If we don't need the "start_matchignore" keyword here, how about just removing it altogether?

**Comment by Eulerizeit** [comment]
Can you explain why we are ignoring things please?

**Comment by 50wu** [comment]
@Eulerizeit Because when we test cases with pg_regress,different distros will print out some different contents,which doesn't affect the results of our test.so we add them into the init_file to ignore.

**Comment by ashwinstar** [comment]
Have we merged the change and PR can be closed or awaiting something ?

**Comment by danielgustafsson** [comment]
It hasn't been merged as far as I can tell, but the file was for an undisclosed reason renamed in 1f6e6f96915fa4ddb1d069a960d8477b990c380a to `gphdfs_init_file` and thus the PR doesn't apply. Since you've hacked on the gphdfs regression suite @adam8157, can you comment on whether we want to apply this still?

**Comment by adam8157** [comment]
Opps, we forgot this one, will resolve the conflicts

**Pull request merged**